main/src/main/java/org/mobilitydata/gtfsvalidator/report/model/FeedMetadata.java

```diff
@@ -309,14 +310,16 @@ private void loadFeedInfo(GtfsTableContainer<GtfsFeedInfo> feedTable) {
     feedInfo.put(
         FEED_INFO_FEED_LANGUAGE, info == null ? "N/A" : info.feedLang().getDisplayLanguage());
     if (feedTable.hasColumn(GtfsFeedInfo.FEED_START_DATE_FIELD_NAME)) {
-      feedInfo.put(
-          FEED_INFO_FEED_START_DATE,
-          info == null ? "N/A" : info.feedStartDate().getLocalDate().toString());
+      LocalDate localDate = info.feedStartDate().getLocalDate();
+      String displayDate =
+          localDate.equals(LocalDate.of(1970, 1, 1)) ? "N/A" : localDate.toString();
+      feedInfo.put(FEED_INFO_FEED_START_DATE, info == null ? "N/A" : displayDate);
     }
     if (feedTable.hasColumn(GtfsFeedInfo.FEED_END_DATE_FIELD_NAME)) {
-      feedInfo.put(
-          FEED_INFO_FEED_END_DATE,
-          info == null ? "N/A" : info.feedEndDate().getLocalDate().toString());
+      LocalDate localDate = info.feedEndDate().getLocalDate();
+      String displayDate =
+          localDate.equals(LocalDate.of(1970, 1, 1)) ? "N/A" : localDate.toString();
```

Comment:
Same comment as before => Let's use the default feed end date set by the Gtfs entity => GtfsFeedInfo.DEFAULT_FEED_END_DATE instead of the LocalDate.of(1970, 1, 1). It is technically the same, but _if_ the default date is changed, this code will likely be left behind.